src/components/PositionedOverlay/tests/PositionedOverlay.test.tsx

```diff
@@ -124,99 +120,6 @@ describe('<PositionedOverlay />', () => {
     });
   });
 
-  describe('intersectionWithViewport', () => {
```

Comment:
This is testing internal details, presumably because it's hard to to test this using just the component.

Move this function to math.ts and do the tests in math.test.ts to avoid exporting `intersectionWithViewport` from PositionedOverlay.tsx